Alekseichik_Dmitry/05/List/List.html

```diff
@@ -0,0 +1,61 @@
+                    <!DOCTYPE html> 
```

Comment:
Could you please align all tags in this file. Looks like someone splashed it everywhere